mailpoet/lib/EmailEditor/Engine/Renderer/Renderer.php

```diff
@@ -85,6 +85,7 @@ public function render(\WP_Post $post, string $subject, string $preHeader, strin
 
     $templateWithContentsDom = $this->inlineCSSStyles($templateWithContents);
     $templateWithContents = $this->postProcessTemplate($templateWithContentsDom);
+    $templateWithContents = $this->preprocessManager->postprocess($templateWithContents);
```

Comment:
Also here `$this->preprocessManager` should probably change as it is no longer only preprocessor.